packages/web-components/src/components/ic-search-bar/ic-search-bar.stories.mdx

```diff
@@ -393,6 +393,83 @@ import readme from "./readme.md";
   </Story>
 </Canvas>
 
+### External filtering with custom loading messages
```

Comment:
Having an issue with this story. When I type "esp" nothing happens. If, after a bit, I add another letter, then I get the loading error. Seems to only be the first time after the story loads though. It would be good to confirm if it is just a storybook issue

Reply:
Oh yeah, I can see that behaviour on the original external filtering with loading example too which hasn't been touched and I think was working before! Might just be a storybook issue I think

Reply:
Has this been looked at and is @ad9242 happy with this response? If so, I'd be happy to approve 😊

Reply:
I checked and there haven't been any changes so it must be a nuance of storybook. I can create a ticket to look into it further if you both think it needs investigating

Reply:
Yes - a ticket to investigate this would be great 👍

Reply:
Ticket created: https://github.com/mi6/ic-ui-kit/issues/1939